frontend/src/pages/Landing/index.js

```diff
@@ -320,11 +324,13 @@ function Landing() {
           setDownloadAlertsError={setDownloadAlertsError}
           downloadAllAlertsButtonRef={downloadAllAlertsButtonRef}
         />
-        <ActivityReportsTable
-          filters={filtersToApply}
-          showFilter={false}
-          tableCaption="Approved activity reports"
-        />
+        <FilterContext.Provider value={{ filters, filterKey: FILTER_KEY }}>
```

Comment:
Three questions:
1) Should this we be putting `filtersToApply` in this context instead of `filters`?
2) I can't remember, why do we need to pass `ActivityReportsTable` `filtersToApply` instead of just filters?
3) If the filters sent to the context matches the prop sent to the `ActivityReportsTable` component (either both `filters` or both `filtersToApply`) can we drop the context altogether?